openpype/hosts/houdini/plugins/create/create_composite.py

```diff
@@ -28,13 +28,13 @@ def create(self, subset_name, instance_data, pre_create_data):
             pre_create_data)  # type: CreatedInstance
 
         instance_node = hou.node(instance.get("instance_node"))
-        filepath = "{}{}".format(
-            hou.text.expandString("$HIP/pyblish/"),
-            "{}.$F4{}".format(subset_name, self.ext)
+        file_path = "{}/pyblish/`chs('subset')`.$F4.{}".format(
+            hou.text.expandString("$HIP"),
+            self.ext
```

Comment:
As explained [here](https://github.com/ynput/OpenPype/pull/5666#issuecomment-1741710742) we're likely better of leaving these to the template system to decide where files should go.

And better yet is that a custom HDA could also then allow to add 'versioning' to the staging outputs as @fabiaserra and others had requested. The HDA could have a "Staging version" integer attribute which could be used, etc.

Reply:
I think we would need to add Houdini vars to template data, 
so that we can have something like this ? 

![image](https://github.com/ynput/OpenPype/assets/20871534/5a3f6a54-117e-4ca9-b52d-6588cbef2ee8)